src/libos/Makefile

```diff
@@ -136,6 +136,29 @@ $(S_OBJS):$(OBJ_DIR)/libos/$(SRC_OBJ)/%.o: src/%.S
 	@$(CC) $(C_FLAGS) -c $< -o $@
 	@echo "AS <= $@"
 
+format: c_format rust_format
+
+c_format: $(C_SRCS) $(CXX_SRCS)
+	@$(C_FORMATTER) $^
+
+rust_format: $(RUST_SRCS)
+	@output=$$(cargo fmt -- --check 2>&1); retval=$$?; \
+		if [[ $$retval -eq 1 ]]; then \
+			$(ECHO) "$$output"; cargo fmt; $(ECHO) "$(GREEN)\nRust code format corrected.$(NO_COLOR)"; \
+		fi
+
+format-check: c_check rust_check
+
+c_check: $(C_SRCS) $(CXX_SRCS)
+	@$(C_FORMATTER) --check $^
+
+rust_check: $(RUST_SRCS)
+	@output=$$(cargo fmt -- --check 2>&1); retval=$$?; \
+		if [[ $$retval -eq 1 ]]; then \
+			$(ECHO) "$(RED)\nRust code format error detected:$(NO_COLOR)"; $(ECHO) "\n$$output"; \
+			$(ECHO) "\nTo get rid of the format warnings above, run $(CYAN)"make format"$(NO_COLOR) to correct"; \
+		fi
```

Comment:
The naming of these targets are not consistent. Some uses `_`, while others use `-`. I suggest using the following names:
```
format
format-c
format-rust

format-check
format-check-c
format-check-rust
```

Reply:
Done.